resources/public/js/stats.js

```diff
@@ -0,0 +1,25 @@
+var ctx = document.getElementById('message-chart');
+
+var messageJson = document.getElementById('message-data').innerHTML;
+var messageData = JSON.parse(messageJson.replace(/&quot;/g,'"'));
+var labels = messageData.map(function(elem) {
+  return elem["day"];
+})
+var data = messageData.map(function(elem) {
+  return elem["msg-count"];
+})
+
+var messageChart = new Chart(ctx, {
+  type: 'line',
+  data: {
+      labels: labels,
+      datasets: [{
+          label: 'No. of slack messages',
+          fill: false,
+          data: data,
+          backgroundColor: 'rgba(255, 99, 132, 0.2)',
+          borderColor: 'rgba(54, 162, 235, 1)',
+          borderWidth: 2
+      }]
+  },
+});
```

Comment:
You're unabashedly checking in JavaScript ?! :scream: 

Just kidding, this is nice and pragmatic. That said anything if it gets any more complicated than this then we should migrate this to ClojureScript.

Reply:
Haha! I knew this would raise some flags, but I had given myself a limit of 1 hour to finish the charts enhancement or stop working on it, and I would've taken a bit longer to setup cljs.

But yes, I'll make an issue so that we can port it to cljs because it *might* get more complicated in the very near feature for stuff like reaction popups / user profile sidebars etc.